doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,137 @@
+
+# PaddlePaddle Client
+PaddlePaddle client is command line tool, you can use a PaddlePaddle client to start a local train and submit a distributed training job to kubernetes cluster.
+
+The relation of PaddlePaddle, kubernetes and docker:
+
+<img src="./submit-job.png" width="500">
+
+
+# Running your training locally
+Execute `paddle local train` to run your local train.
+```bash
+paddle local train
+  --pcakage-path=./demo
+  --module=demo.train
+  --input=<input_dir>
+  --output=<output_dir>
+  --image=<paddle_image>
+  --e=NUM_PASS=4
+```
+- package-path: your trainer code python package
+- module: include a main function, trainer entrance.
+- input: input directory, for local train, it's a host path.
+- output: output directory, for local train, it's a host path.
+- image: paddlepaddle production image
+- e: environment varible
+
+When you start the local train, the client starts a docker container like:
+```bash
+  docker run --rm
+  -v <input_dir>:/train/input
+  -v <output_dir>:/train/output
+  -v <package-path>:/train/package
+  -e NUM_PASS=4 <paddle_image>
+  python /train/package/train.py
+```
+
+
+# Submit distributed training job
+You can use `paddle submit job <job-name>` to submit a distributed training job.
+
+```bash
+paddle job submit train <job-name>
+  --package-path=/train/quick_start
+  --module=quick_start.train
+  --input=<input_dir>
+  --output=<output_dir>
+  --trainers=4
+  --pservers=2
+  --image:<your image>
+  -e=NUM_PASS=5
+```
+
+- job-name: you should specify a unique job name,
+- package-path=your python package files
+- module: include the main function, trainer entrance
+- input: input directory on distributed file system
+- output: output directory on distributed file system
+- trainers: trainer process count
+- pserver: parameter process count
+- image: your trainer docker image, include your trainer files and dependencies.
+- command:
+- e: environment variable
+
+## Build your docker image
+Before submitting a distributed training, you should build your docker image, here
```

Comment:
What I expected is that `paddle train` pack everything into a Docker image, other than users pack them?